tv_series.md

```diff
@@ -25,54 +25,66 @@
   - [Chuck](#chuck)
 - [7.5 :star:](#_75)
   - [The Lone Gunmen](#the-lone-gunmen)
+- [7.3 :star:](#_73)
+  - [Bandersnatch: A Black Mirron Event](#bandersnatch)
 - [6.6 :star:](#_66)
   - [ .hack//Roots](#hack_roots)
 
 ### 8.9
 
 #### Black Mirror
+
 > A television anthology series that shows the dark side of life and technology.
 
 ![Black_Mirror](assets/black_mirror.jpg)
-* Year: 2011-
-* Country: UK
-* [_IMDb_](https://www.imdb.com/title/tt2085059/)
+
```

Comment:
too lot of changes.. Why?

Reply:
I just added the Bandersnatch that's all, nothing else changed

Reply:
seems you've added spaces after every header.